src/tab/header/item/TabHeaderItem.tsx

```diff
@@ -29,6 +30,7 @@ function TabHeaderItem({testid, tab, onClick, index, isActive}: TabHeaderItemPro
   );
 
   function handleClick() {
+    if (tab.isDisabled) return;
```

Comment:
We prefer to have curly braces when defining if blocks
```suggestion
    if (tab.isDisabled) {
      return;
    }
```